README.md

```diff
@@ -39,12 +39,12 @@ Add the dependency:
 <dependency>
   <groupId>com.evolutiongaming</groupId>
   <artifactId>prometheus-cassandra-driver</artifactId>
-  <version>0.7</version>
+  <version>1.0</version>
 </dependency>
 ```
 * SBT:
 ```scala
-libraryDependencies += "com.evolutiongaming" % "prometheus-cassandra-driver" % "0.7"
+libraryDependencies += "com.evolutiongaming" % "prometheus-cassandra-driver" % "1.0"
```

Comment:
Please add a section to README about prometheus client version compatibility - 0.7 should be marked as the last version working with pre-0.10.0, starting from 1.0 the client has to be 0.10.0+

Reply:
added